### ISSUE TITLE
martian: server sent events without chunked encoding

### ISSUE THREAD

**Issue by mmatczuk** [saucelabs/forwarder#154]
Our support for Server Sent Events is based on chunking.
Chunking in SSE isn't mandatory it can be also harmful and is not recommended.

> Authors are also cautioned that HTTP chunking can have unexpected negative effects on the reliability of this protocol, in particular if the chunking is done by a different layer unaware of the timing requirements. If this is a problem, chunking can be disabled for serving event streams.

Source https://html.spec.whatwg.org/multipage/server-sent-events.html#authoring-notes

In NGINX the SSE support requires the following configuration

```
proxy_set_header Connection '';
proxy_http_version 1.1;
chunked_transfer_encoding off;
```

Source https://stackoverflow.com/questions/13672743/eventsource-server-sent-events-through-nginx

Events can be [separated by](https://html.spec.whatwg.org/multipage/server-sent-events.html#parsing-an-event-stream) `cr lf` or `cr` or `lf`.

**Comment by mmatczuk** [comment]
The scope is limited to HTTP servers without chunking. It will be more important when we add MITM support.

**Comment by Choraden** [comment]
Fixed in #812
